oldeditablebuffer.go

```diff
@@ -10,22 +10,22 @@ import (
 	"github.com/rjkroege/edwood/internal/file"
 )
 
-// File is an editable text buffer with undo. Many Text can share one
-// File (to implement Zerox). The File is responsible for updating the
-// Text instances. File is a model in MVC parlance while Text is a
+// OldEditableBuffer is an editable text buffer with undo. Many Text can share one
+// OldEditableBuffer (to implement Zerox). The OldEditableBuffer is responsible for updating the
+// Text instances. OldEditableBuffer is a model in MVC parlance while Text is a
 // View-Controller.
 //
-// A File tracks several related concepts. First it is a text buffer with
+// A OldEditableBuffer tracks several related concepts. First it is a text buffer with
 // undo/redo back to an initial state. Mark (undo.RuneArray.Commit) notes
 // an undo point.
 //
-// Next, a File might have a backing to a disk file.
+// Next, a OldEditableBuffer might have a backing to a disk OldEditableBuffer.
 //
 // Lastly the text buffer might be clean/dirty. A clean buffer is possibly
 // the same as its disk backing. A specific point in the undo record is
 // considered clean.
 //
-// TODO(rjk): File will be a facade pattern composing an undo.RuneArray
+// TODO(rjk): OldEditableBuffer will be a facade pattern composing an undo.RuneArray
```

Comment:
This TODO is outdated. `OldEditableBuffer` will not be all of this goodness. For that, we have the (glorous 😀) `file.EditableBuffer`. Please update.

Reply:
+1 vote to rename file.EditableBuffer to file.GlorousEditableBuffer